### ISSUE TITLE
Question about google/bigbird-pegasus-large-pubmed

### ISSUE THREAD

**Issue by tetsu9923** [huggingface/transformers#16777]
Hi, I have a question about the BigBird seq2seq model [google/bigbird-pegasus-large-pubmed](https://huggingface.co/google/bigbird-pegasus-large-pubmed).
How was this model trained?
Was it pretrained with just word-level or sentence-level masking?
Or was it further fine-tuned by the summarization task?
Thank you.

**Comment by NielsRogge** [comment]
From the model card:

> This checkpoint is obtained after fine-tuning BigBirdPegasusForConditionalGeneration for summarization on pubmed dataset from [scientific_papers](https://huggingface.co/datasets/scientific_papers)[](https://huggingface.co/google/bigbird-pegasus-large-pubmed#bibtex-entry-and-citation-info).

Also, please use our [forum](https://discuss.huggingface.co/) for such questions. We'd like to keep Github issues for bugs/feature requests.

Thanks!
